### PR TITLE
Restore some of the symfony fixers

### PR DESCRIPTION
**Pull request by greg0ire**
Some of them are really good and handy, let's use them!

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
You should add concat_with_space. You code is already like this.

Plus, this make sense because it's like a string interpolation comparing to the classic operators.

**Comment by Soullivaneuh** [comment]
Also useless else?

File: .styleci.yml [hunk 1]
```diff
@@ -1,8 +1,20 @@
 preset: psr2
 
 enabled:
+  - concat_with_spaces
```
Comment (Soullivaneuh) [inline .styleci.yml]: Sorry, I would say `without` in https://github.com/greg0ire/enum/pull/43#issuecomment-227983772.

Comment (greg0ire) [inline .styleci.yml]: Ooooh ok.

**Comment by Soullivaneuh** [comment]
You should restrict branches on Travis to avoid this kind of things:

image

**Comment by greg0ire** [comment]
Didn't know you could do that :)

**Comment by Soullivaneuh** [comment]
This exactly because of this I did it on Sonata projects! :wink:

**Comment by greg0ire** [comment]
See #48

**Pull request merged**